substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/ClassEntry.java

```diff
@@ -71,12 +72,20 @@ public class ClassEntry extends StructureTypeEntry {
      */
     private Map<ResolvedJavaMethod, MethodEntry> methodsIndex;
     /**
-     * A list recording details of all primary ranges included in this class sorted by ascending
-     * address range.
+     * A list recording details of all normal primary compiled methods included in this class sorted
+     * by ascending address range. Note that the associated address ranges are disjoint and
+     * contiguous.
      */
     private List<PrimaryEntry> primaryEntries;
```

Comment:
Minor: I believe renaming this to `normalPrimaryEntries` will make more clear what this list is about (or at least hint the readed to check the javadoc :) ).

Reply:
I have a follow up patch which includes that change. The follow-up also renames class `PrimaryEntry` to `CompiledMethodEntry`, adjusts parameters and locals to use the name `compiledEntry` or instead of `primaryEntry` and updates comments to use the phrase `compiled method` instead of `primary method`.

Reply:
This renaming patch has now been pushed